src/Spaces/finitedifference.jl

```diff
@@ -161,7 +161,12 @@ The index of the z-component of an abstract point
 in an `AxisTensor`.
 """
 z_component(::Type{<:Geometry.LatLongZPoint}) = 9
```

Comment:
should this be deltaz component?

Reply:
I'm fine with renaming it if you thing `dz_component` is a better name. It's used in `getproperty(lg.∂x∂ξ.components.data, z_component(data_layout_type))`, so I interpreted that as the `z`-component of `dx` (the multi-dimensional vector)

Reply:
`Δz_metric_component`